CRM/Core/Payment/Manual.php

```diff
@@ -96,7 +96,7 @@ public function getPaymentFormFields() {
       // Really we should render check_number here, but we need to review how we edit
```

Comment:
I assume that we are going to be rendering check number properly on payment area of pages, not in overal contribution area. Since we are addressing the implied To Do of this comment, let's remove the comment.